doc/rados/configuration/ceph-conf.rst

```diff
@@ -178,8 +190,8 @@ on the same source (i.e. that is, in the same configuration file), the
 If multiple values of the same configuration option are specified in the same
 section, the last value specified takes precedence.
 
-Note that values from the local configuration file always take precedence over
-values from the monitor configuration database, regardless of the section in
+Note that values set in the local configuration file always take precedence over
+values from the Monitor central configuration database, regardless of the section in
```

Comment:
s/Monitor central/Monitor's central/

I think this is a little clearer, but I am open to being convinced otherwise.